near2eth/near2eth-block-relay/index.js

```diff
@@ -186,10 +188,53 @@ class Near2EthRelay {
     const maxDelay = Number(near2ethRelayMaxDelay)
     const errorDelay = Number(near2ethRelayErrorDelay)
 
+    const selectDuration = web3.utils.toBN(Number(near2ethRelaySelectDuration) * 1000_000_000)
```

Comment:
Let's move this variable closer to its usage and also please add time unit suffix in the name